templates/repo/icon.tmpl

```diff
@@ -5,6 +5,8 @@
 	{{svg "octicon-mirror" 24}}
 {{else if $.IsFork}}
 	{{svg "octicon-repo-forked" 24}}
+{{else if $.IsPrivate}}
```

Comment:
But there is already a label indicating the repository is private.

Reply:
Both the `.IsPrivate` and `.IsTemplate` template conditions render labels and I guess for clarity we should keep these labels. I think it's not a big issue if the repo icon now additionally displays these states.

Reply:
resolved per discussion below